data/migrations/V0100__add_ofec_sched_a_agg_state_vw.sql

```diff
@@ -0,0 +1,19 @@
+DROP VIEW IF EXISTS public.ofec_sched_a_agg_state_vw;
+
+CREATE OR REPLACE VIEW public.ofec_sched_a_agg_state_vw AS
```

Comment:
Did you miss my earlier comment?
> Can we make the layout easier to read and more consistent? I don't know if we ever completed the work @hcaofec was doing on database code conventions. Let's start with uppercase keywords, lowercase for all else, indented SELECT list, indented JOIN conditions, etc.

Reply:
Sorry, I didn't see your previous comment.
 good point, I will check the database code conventions with @hcaofec @fecjjeng . Thanks